docs/multiple_repos.md

```diff
@@ -1,16 +1,160 @@
 ---
-tilt: Structuring Multiple Repos
+title: Many Tiltfiles and Many Repos
 layout: docs
 sidebar: guides
 ---
 
-# Structuring Multiple Repos
-When you run `tilt up` in a terminal, Tilt looks for a Tiltfile in that directory, parses it, and typically starts multiple services rooted in that directory. Usually that directory is one git source-controlled repo. But what if you you need to run multiple services, spread across multiple repos, during development? Tilt supports that too.
+Most of our example projects use a single Tiltfile. All the source files,
+Dockerfiles, and Kubernetes configs are in a single folder.
 
-## Entrypoint Tiltfile
-First create a Tiltfile for each repo that has resources of interest. Then, have an "entrypoint" repo with a Tiltfile that points to other Tiltfiles in other repos. Typically this would be in a parent directory. You then run Tilt in the entrypoint repo, with logic that starts resources in those other repos. In particular, use [`include()`](api.html#api.include) or [`load()`](api.html#api.load) to load those resources.
+But your project may be organized differently!
 
-Each team or developer may have different needs in starting specific services, especially if your entrypoint Tiltfile is pointing to many downstream Tiltfiles. Consider using [Tiltfile Configs](tiltfile_config.html) to help customize accordingly.
+## How to Organize Your Project for Tilt
 
-If you have Tiltfile logic that can be abstracted out of your specific system, consider using [Extensions](extensions.html).
+Tilt does not require a particular project structure. We try to provide 
+APIs to help you pull the services you need into your dev environment, no matter
+where they're defined.
 
+Do NOT restructure your project for Tilt. Instead, restructure your Tiltfiles
+to match your project structure!
```

Comment:
```suggestion
>> Do NOT restructure your project for Tilt. Instead, restructure your Tiltfiles to match your project structure!
```
Suggest to put this in a blockquote to make it more visible

![Screenshot 2022-02-16 at 13 36 53](https://user-images.githubusercontent.com/2743542/154266366-b1d73c02-5222-4fd4-a0cf-1abbda6b9b0c.png)